projects/aca-shared/src/lib/adf-extensions/extensions-data-loader.guard.ts

```diff
@@ -75,4 +75,8 @@ export class ExtensionsDataLoaderGuard implements CanActivate {
       })
     );
   }
+
+  canActivateChild(route: ActivatedRouteSnapshot): Observable<boolean> {
```

Comment:
I'm not sure it is needed. Wouldn't this result to load always on every child route? We want to load these loaders only once, not on every child routes.